webapp/channels/src/components/suggestion/at_mention_provider/at_mention_provider.tsx

```diff
@@ -342,15 +398,17 @@ export default class AtMentionProvider extends Provider {
         this.updateMatches(resultCallback, this.items());
 
         // If we haven't gotten server-side results in 500 ms, add the loading indicator.
-        let showLoadingIndicator = setTimeout(() => {
+        let showLoadingIndicator: NodeJS.Timeout | null = setTimeout(() => {
             if (this.shouldCancelDispatch(prefix)) {
                 return;
             }
 
-            this.updateMatches(resultCallback, this.items().concat([{
+            this.updateMatches(resultCallback, [...this.items(), ...[{
                 type: Constants.MENTION_MORE_MEMBERS,
                 loading: true,
-            }]));
+            }]]);
+
+            this.items().concat();
```

Comment:
why do we need this?

Reply:
i might have forgot to remove that ! will remove that !